docs/pages/guides/mcp.mdx

```diff
@@ -0,0 +1,179 @@
+    alt="Install MCP Server"
+    className="hidden max-w-48 dark:block"
+  />
+  <img
+    src="https://cursor.com/deeplink/mcp-install-dark.svg"
+    alt="Install MCP Server"
+    className="block max-w-48 dark:hidden"
+  />
+</a>
+
+</Collapsible>
+
+<Collapsible summary="VS Code setup">
+
+1. Open Command Palette (<kbd>Cmd ⌘</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> or <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>)
+2. Run **MCP: Add Server**
+3. Select **HTTP**
+4. Enter the server details:
+   - **URL**: `https://mcp.expo.dev/mcp`
+   - **Name**: expo-mcp
+
+</Collapsible>
+
+</Step>
+
+<Step label="2">
+
+### Authenticate with Expo
+
+After installing the MCP server, you'll need to authenticate using one of two methods:
+
+#### Access token (recommended)
+
+Generate an access token from your Expo account settings and use it during the OAuth flow.
+
+#### Credentials
+
+Use your Expo account username and password. In this case, the server will generate an access token automatically.
+
+</Step>
+
+<Step label="3">
+
+### Set up local capabilities (Recommended)
+
+> **info** Local capabilities are only available in **SDK 54 and above**.
+
+For the full MCP experience with advanced features like taking screenshots from your iOS Simulator, opening DevTools, and automation capabilities, set up a local Expo development server:
+
+<Terminal
+  cmd={[
+    '$ cd /path/to/your-project',
+    '',
+    '# Install expo-mcp package',
+    '$ npx expo install expo-mcp --dev',
+    '',
+    '# Start dev server with MCP capabilities',
+    '$ EXPO_UNSTABLE_MCP_SERVER=1 npx expo start',
+  ]}
+/>
+
+> **important** Whenever you start or stop the development server, you need to **reconnect or restart** your MCP server connection in your AI tool to ensure the AI tool gets refreshed capabilities.
+
+</Step>
+
+## Server Capabilities vs Client Capabilities
+
+The Expo MCP server provides two types of capabilities depending on your setup:
+
+### Server Capabilities
+
+Server capabilities are available with just the remote MCP server connection, without needing to set up a local development server. The **generate_agents_md** tool is an example of a server capability.
+
+### Client Capabilities
+
+Client capabilities require a local Expo development server to be running and provide advanced features that interact with your local development environment:
+
+- **Automation tools**: Take screenshots, tap views, find elements by testID
+- **Development tools**: Open React Native DevTools
+- **Project analysis**: Generate expo-router sitemap
+
+These capabilities enable more sophisticated workflows like automated testing, visual verification, and deeper project introspection.
+
+## Available MCP Capabilities
+
+### Tools
+
+| Tool                                   | Description                                                          | Example Prompt                                      | Availability                 |
```

Comment:
available tools are subject to change. we can review all the other part in the doc first